src/components/searchForm.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import InputWithAutocomplete from './inputWithAutocomplete';
+
+class SearchForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            inputText: '',
+            isSubmitable: false
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(value) {
+        this.setState({
+            inputText: value,
+            isSubmitable: this.isValid(value)
+        });
+    }
+
+    isValid(part) {
```

Comment:
@yarostbaklajana 
`location.name.toUpperCase().startsWith(part.toUpperCase())).slice(0, 10).find((hint) => hint.name === part);`
could be replaced with
`location,name === part` and than check that filtered array length >= 1